### ISSUE TITLE
Move the weapons policy section

### ISSUE THREAD

**Issue by jenlampton** [backdrop-ops/conduct#2]
I'm not 100% sold on the weapons policy bit, it feels a little out of place between `Unacceptable Behavior` and `Consequences of Unacceptable Behavior`. Maybe we just move it somwehere else in the document?

**Comment by yorkshire-pudding** [comment]
@jenlampton I agree it kind of clangs where it is.  I wonder whether we need to understand if this is referring just to physical events or would also apply to someone having weapons on display during a video call.  

Some of the 'sexual' bullet points within unacceptable behaviour relate to physical meetings of the community or parts thereof, whereas others apply to anywhere; so perhaps there should be common policies for all situations and then specific online (e.g. you must be dressed before appearing on video) and physical meeting policies, of which weapons and physical sexual conduct would be part?
